### ISSUE TITLE
Thead hiding first rows

### ISSUE THREAD

**Issue by Kaari** [mkoryak/floatThead#251]
Using EmberJS to fill the table,
The thead is located on top of the first rows, which hides them.

Screenshot:
(without any css styles, it's the same behavior)
(the 3 rows you're seeing are the 3 first rows of the table)
![2015-09-20 tble content hidden](https://cloud.githubusercontent.com/assets/6136909/9979872/e55e7db6-5f82-11e5-9a1a-aef5ed23af68.png)

Code:
```
didInsertElement: function () {
  this.$('table.apps').floatThead();
  this.get('puppetDB').getFacts(); // fill the table Ember-way (adding rows to the model)
}
```

The really buggy thing I spotted is when I'm trying to use `useAbsolutePositioning`:
With `{useAbsolutePositioning:false}`: nothing is changing.
With `{useAbsolutePositioning:true}` : Initially, the result is the same but if I'm scrolling down on the page, and then scrolls up to the top, the headers is placing well, where it should be.

By the way, I'm thinking at least one thing is buggy, I have not the same behavior with `{useAbsolutePositioning:true}` than without specifying this property, isn't `true` the default value for this property ?

Thanks.
Please ask if u need more infos.

**Comment by mkoryak** [comment]
its really hard to help you without seeing your code. I do know that you should look  at the docs for the following 2 options:

`reflow` method and `headerCellSelector`

**Comment by Kaari** [comment]
Thank you very much for your answer, I did not spotted `headerCellSelector` before, and actually my <thead> had no <tr> into, which caused this bug.
By the way, I managed to avoid using `reflow` by loading the data before html rendering, which solved another issue I encountered, the reflow is rly useful when dealing with DOM modification, and I think the auto-reflow you're working on is a rly good idea to provide a component natively supported by any js client framework.

Thanks you for your work, I wish you good luck ;).
